From c935053e346fc01c477f39049e122dd5114f3f0c Mon Sep 17 00:00:00 2001
From: Thomas A Caswell <tcaswell@gmail.com>
Date: Sun, 4 Oct 2015 15:51:22 -0400
Subject: FIX: remove incorrect error checking

Currently fails with qt5
Patch-Name: 10ef9a7e26dc34328b7ec7fd0ed2c2db6104b342.patch
---
 lib/matplotlib/backends/qt_compat.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lib/matplotlib/backends/qt_compat.py b/lib/matplotlib/backends/qt_compat.py
index 6ce8fd6..326f0f6 100644
--- a/lib/matplotlib/backends/qt_compat.py
+++ b/lib/matplotlib/backends/qt_compat.py
@@ -140,11 +140,6 @@ if _sip_imported:
             def _getSaveFileName(*args, **kwargs):
                 return QtGui.QFileDialog.getSaveFileName(*args, **kwargs), None
 
-    else:
-        raise RuntimeError("PyQt{4,5} bindings found despite sip importing\n"
-                           "Please install PyQt4 or PyQt5, uninstall sip or "
-                           "explicitly set the pyside backend.")
-
     # Alias PyQt-specific functions for PySide compatibility.
     QtCore.Signal = QtCore.pyqtSignal
     try:
